skizze-cli - Pressing "Enter" crashes skizze-cli

Pressing "Enter" (without typing any command) on skizze-cli throws and index out of range:

```
panic: runtime error: index out of range

goroutine 1 [running]:
skizze-cli/bridge.evalutateQuery(0xc8200efc40, 0x0, 0x0, 0x0)
	/Users/martinpinto/go/src/github.com/seiflotfy/skizze/src/skizze-cli/bridge/loop.go:62 +0xc83
skizze-cli/bridge.Run()
	/Users/martinpinto/go/src/github.com/seiflotfy/skizze/src/skizze-cli/bridge/loop.go:127 +0x28d
```
The problem apparently lies where queries are being handled (src/skizze-cli/bridge/loop.go:62):
```
if len(fields) <= 2 {
		//TODO: global stuff might be set
		switch strings.ToLower(fields[0]) {
```
**Steps to reproduce:**

1. Start skizze-cli
2. Press enter on skizze-cli
3. --> skizze-cli crashes: index out of range

--- a/src/skizze-cli/bridge/loop.go
+++ b/src/skizze-cli/bridge/loop.go
@@ -57,7 +57,7 @@ func getFields(query string) []string {
 
 func evalutateQuery(query string) error {
 	fields := getFields(query)
-	if len(fields) <= 2 {
+	if len(fields) != 0 && len(fields) <= 2 {
 		//TODO: global stuff might be set
 		switch strings.ToLower(fields[0]) {
 		case "list":

